Adding custom URL handling

aidenmitchell opened the pull request:
I've never contributed in Python before, so any feedback is appreciated :)

aidenmitchell commented:
Note: From my understanding, this would make `-m` no longer required, assuming `-u` is provided. Not sure how that should be handled.

aidenmitchell commented:
It may also be worth noting somewhere to the user that:
1. Having HTTP or HTTPS in their URL is required
2. The full path to the payload must also be specified

I did add this, for point 1:
https://github.com/aidenmitchell/follina.py/blob/4dfab2d5ee1c087f31d6a168a310a71a44bbd9a8/follina.py#L53-L56=

chvancooten commented:
Hey, thanks for the PR! This looks good for users that want to use e.g. a reverse proxy to serve their payload. Perhaps we should clarify though that a user-specified URL option does _not_ reflect on the web server by definition, so that differences in port/protocol should be handled by the user itself.
Added a couple of comments, unfortunately don't have time to pick it up myself today.

aidenmitchell commented:
Sounds good, I've removed the parsing and added clarity to the help text:
https://github.com/chvancooten/follina.py/blob/7550505777e4e0e3f2e4fbe439797b1d592db35f/follina.py#L36
